### ISSUE TITLE
[BITV] 9.1.1.1c/3.1 - The "Details" area / "Versions" tab - Both "preview" images and "restore" images do not have the alt attribute. To designate images as decorative images, the attribute alt="" should be used. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36910]
- [x] Fix
- [ ] Backport

<img width="378" alt="2023-03-20 13_25_27-Photos - Files - Nextcloud – Mozilla Firefox" src="https://user-images.githubusercontent.com/1315170/226338455-0874466a-20f5-4641-b4d9-8ca321cef70e.png">

example:
```
<img class="preview" src="/ltd/a11y/index.php/apps/files_versions/preview?file=%2FPhotos%2FReadme.md&amp;version=1675688665" width="44" height="44">
```
```
<a href="#" class="revertVersion" title="Restore"><img src="/ltd/a11y/core/img/actions/history.svg"></a>
```

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3a2875951a-v3-n1

</details>

- [ ] 9\.1.1.1c/10.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3a2875951a-v10-n1

**Comment by JuliaKirschenheuter** [comment]
For NC26+ is nothing to do, already implemented. 

**Note:In Chrome an empty `alt` attr won't be shown right! Double check in (for example) Firefox:**

Firefox:
![Screenshot from 2023-03-23 16-13-14](https://user-images.githubusercontent.com/6078378/227249136-893169b9-361c-48c8-a115-c70c51906a83.png)

Chrome: 
![Screenshot from 2023-03-23 16-16-14](https://user-images.githubusercontent.com/6078378/227249343-b9374cf0-8243-4e62-9c34-c6f3ab2fcc63.png)

**Comment by JuliaKirschenheuter** [comment]
Fix:
- [ ] https://github.com/nextcloud/server/pull/37363

Backports:
